<h2>Обработка исключений</h2>

<v-code lang="js" title="">
try {
	// инструкции, которые могут бросить исключение
} catch (идентификатор) {
	// инструкции будут выполняться если в try произошло исключение
	// передается выражение (идентификатор), которое было в throw
} finally {
	// инструкции будут выполняться независимо от исключения
}
// catch или finally может отсутствовать
// если ошибка необработана, скрипт не будет выполняться
// если ошибка обработана в catch, скрипт продолжит выполнение
</v-code>

<v-code lang="js" title="">
try {
	throw new Error('Panic!');
} catch (e) {
	console.log(e.message);
} finally {
	console.log('Код, который нужно выполнить');
}
</v-code>
